src/components/elements/accordion/types.ts

```diff
@@ -21,6 +21,7 @@ export interface IAccordionSchema<V = undefined, C = undefined>
 	expanded?: boolean | undefined;
 	displayState?: BoxContainerDisplayState | undefined;
 	title: string;
+	titleSchema?: Record<string, TFrontendEngineFieldSchema<V, C>> | undefined;
```

Comment:
restrict the elements supported in title

```suggestion
	titleSchema?: Record<string, ITextSchema | IPopoverSchema | IInlineWrapperSchema> | undefined;
```

Reply:
udpated